csfieldguide/chapters/content/en/computer-graphics/sections/drawing-lines-and-circles.md

```diff
@@ -1,6 +1,6 @@
 # Drawing lines and circles
 
-A fundamental operation is computer graphics is to draw lines and circles.
+A fundamental operation in computer graphics is to draw lines and circles.
 For example, these are used as the components of scalable fonts and vector graphics;
 the letter "g" is specified as a series of lines and curves,
 so that when you zoom in on it the computer can redraw it at whatever resolution is needed.
```

Comment:
The comment on line 9 is shown to the user.
![Screen Shot 2019-04-09 at 1 14 10 PM](https://user-images.githubusercontent.com/16066822/55766597-7935b500-5ac9-11e9-9603-9a37baf38960.png)